src/main/java/racing/Application.java

```diff
@@ -1,12 +1,25 @@
 package racing;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
 public class Application {
 
     public static void main(String[] args) {
+        //참가 자동차 목록 입력
+        Output.carNameInput();
+        Cars cars = Input.carsInput();
+        Output.carNameOutput(cars);
+
+        //시행 횟수 입력
+        Output.countQuestion();
+        Count count = Input.count();
+        Output.count(count);
+        System.out.println();
+
+        //레이싱 시작
+        Game.game(cars, count);
 
+        //우승자 시상
+        Cars winner = GameController.winner(cars);
```

Comment:
잘하셨는데, `cars`가 이미 필요한 정보를 모두 갖고 있으니 `cars`에게 우승자 목록을 달라고 요청하면 더 좋을 것 같아요.
```suggestion
        Cars winner = cars.getWinners();
```